cli/app/app.go

```diff
@@ -253,3 +256,10 @@ func askForConfirmation() bool {
 		return askForConfirmation()
 	}
 }
+
+func unknownArgs(c *cli.Context) {
```

Comment:
my preference would be to rename this function to `TestForUnknownArgs` or `CheckForUnknownArgs`